src/v2/Apps/Order/Components/PaymentMethodSummaryItem.tsx

```diff
@@ -22,6 +25,15 @@ const PaymentMethodSummaryItem = ({
   return (
     <StepSummaryItem {...others}>
       <CreditCardDetails {...cardInfoWithTextColor} />
+
+      {/* TODO: when we get paymentMethod from API, check against it to render the corrent payment summary */}
+      {/* {paymentMethod === PaymentMethods.CreditCard ? (
+        <CreditCardDetails {...cardInfoWithTextColor} />
+      ) : paymentMethod === PaymentMethods.BankDebit ? (
+        <div>it is bank transfer</div>
+      ) : (
+        <WireTransferDetails />
+      )} */}
```

Comment:
this is the conditional where we'll be checking the payment method of the order (when our API provides it) and render the appropriate payment summary component.